docs/_panels/jenkins-job-categories.md

```diff
@@ -32,7 +32,8 @@ contains the category name we are looking for.
 From left to right and top to bottom, the metrics provided are:
 
 * **Big Numbers**: total numbers about what we are 
-  visualizing in the panel. 
+  visualizing in the panel.
+* **Jenkins projects**: a pie chart that shows the projects that include the different Jenkins instances.
```

Comment:
I'd rename it to Projects and Jobs and would count jobs instead of builds to align results with the dashboard's main use case.